proto/message.py

```diff
@@ -374,8 +382,8 @@ def __init__(self, mapping=None, **kwargs):
 
     def __bool__(self):
         """Return True if any field is truthy, False otherwise."""
-        return any([k in self and getattr(self, k)
-                    for k in self._meta.fields.keys()])
+        return any(k in self and getattr(self, k)
+                   for k in self._meta.fields.keys())
 
     def __contains__(self, key):
         """Return True if this field was set to something non-zero on the wire.
```

Comment:
And, in fact, this is precisely how `__contains__` is advertised here.

Reply:
Actually, reading the code, I think it even _already works_.

Reply:
Yup. Turns out I reinvented a wheel :P